src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/TranscriptListItem.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect, useDispatch } from 'react-redux';
+import { FormattedMessage, injectIntl } from '@edx/frontend-platform/i18n';
+import {
+  Card, Dropdown, Icon, IconButton,
+} from '@edx/paragon';
+
+import { MoreVert } from '@edx/paragon/icons';
+
+import LanguageSelect from './LanguageSelect';
+import hooks from './hooks';
+import { actions } from '../../../../../../data/redux';
+import FileInput from '../../../../../../sharedComponents/FileInput';
+import messages from './messages';
+
+export const TranscriptListItem = ({
+  title,
+  language,
+  // redux
+  downloadTranscript,
+  deleteTranscript,
+}) => {
+  const fileInput = hooks.fileInput({ onAddFile: hooks.replaceFileCallback({ language, dispatch: useDispatch() }) });
+
+  return (
+    <Card>
+      <Card.Header
+        subtitle={title}
+        actions={(
+          <Dropdown>
+            <Dropdown.Toggle
+              id="dropdown-toggle-with-iconbutton-video-transcript-widget"
+              as={IconButton}
+              src={MoreVert}
+              iconAs={Icon}
+              variant="primary"
+              alt="Actions dropdown"
+            />
+            <Dropdown.Menu className="video_transcript Action Menu">
+              <Dropdown.Item
+                key={`transcript-actions-${title}-replace`}
+                onClick={fileInput.click}
+              >
+                <FormattedMessage defaultMessage={messages.replaceTranscript} />
```

Comment:
Change all dropdown menu items to 
```
<FormattedMessage { ...messages.{messageName} />

```
Currently getting errors because the id and description are not being added to the component.

Reply:
Done